synapse_data_warehouse/synapse_raw/tables/R__projectsettingsnapshots_append_task.sql

```diff
@@ -0,0 +1,35 @@
+use role accountadmin;
```

Comment:
Could we try removing this row?

Reply:
Hmm sure, this is role is going off the template in the Contributions page, though. Wouldn't it give us an error without it? (I shut down my codespace so I can't test this super fast)
<img width="830" alt="image" src="https://github.com/Sage-Bionetworks/snowflake/assets/32107699/c1bfae7e-d6cb-483a-9900-feeccab5e088">

Reply:
@jaymedina . so... the should be updated, but basically, we want to avoid creating anything with the Accountadmin role